apps/web/pages/api/proxy/index.ts

```diff
@@ -0,0 +1,70 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
```

Comment:
Ideally should be in the `app/` directory following App Router spec since we want to move in that direction.